emperor/core.py

```diff
@@ -278,6 +291,31 @@ def make_emperor(self, standalone=False):
         # avoid unicode strings
         coord_ids = list(map(str, self.mf.index.tolist()))
 
+        # TODO: This will be removed once the custom axes creation is moved to
+        # the graphical user interface i.e. to the Axes tab.
+        if custom_axes:
```

Comment:
Kinda, you will always be able to select up to 3 custom axes, but you will be able to choose from however many are present in your coordinates. One thing I should have clarified before is that when we add a custom axis, we still preserve (unless otherwise specified) at least 5 dimensions of the PCoA matrix, so that means that irregardless of the number of custom axes, users can always switch to those other axes with the `Axes` tab.